webapp/tests/forms/steps/test_eligibility_steps.py

```diff
@@ -3216,19 +2908,6 @@ def correct_session_data_without_registration_method(self):
             'other_income_eligibility': 'no'}
         return correct_session_data_without_registration_method
 
-    @pytest.fixture
-    def correct_session_data_with_registration_method_software(self, correct_session_data_without_registration_method):
-        correct_session_data_with_registration_method_software = {
-            **correct_session_data_without_registration_method,
-            **{'elster_registration_method_eligibility': 'software'}}
-        return correct_session_data_with_registration_method_software
-
-    @pytest.fixture
-    def correct_session_data_with_registration_method_unknown(self, correct_session_data_without_registration_method):
-        correct_session_data_with_registration_method_unknown = {
-            **correct_session_data_without_registration_method,
-            **{'elster_registration_method_eligibility': 'unknown'}}
-        return correct_session_data_with_registration_method_unknown
 
     def test_if_post_and_session_data_correct_without_registration_method_and_input_data_correct_then_set_success_step(self, app, correct_session_data_without_registration_method):
```

Comment:
Could you search of the strings `registration_method` and `abrufcode` in the webapp to make sure we do not have dangling tests like this one depending on those fields (without actually setting them, thus not creating an error)?